hs_core/models.py

```diff
@@ -1521,8 +1534,60 @@ def update(self, metadata):
                             self.identifiers.filter(name=id_item[element_name]['name']).delete()
                             self.create_element(element_model_name=element_name, **id_item[element_name])
 
+
+    # create crossref deposit xml for resource publication
+    def get_crossref_deposit_xml(self, pretty_print=True):
```

Comment:
Not sure why this function is part of the ```CoreMetaData``` class? I think it should be a function in ```BaseResource``` class.